webpack/graphql/queries/variableOverrides.gql

```diff
@@ -0,0 +1,21 @@
+query($id: String!, $hostId: Int!) {
+  host(id: $id) {
+    allAnsibleRoles {
+      nodes {
```

Comment:
I might be missing some graphql background here, why do we need the extra `nodes` layer here and below?

Reply:
It is because of Relay style connections that add additional information such as total count of the nodes.
```
allAnsibleRoles {
  totalCount
  pageInfo {
     hasNextPage
  }
  nodes {
   //...
  }
}
```

Reply:
Do we need this additional information here? if I'm not mistaken, all variables are loaded without pagination and we don't use the count anywhere.

Reply:
never mind, I played with it a bit and now I understand why this is needed.